go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -874,3 +875,10 @@ func InStatement(yield Query) {
 		yield(fmt.Sprintf("%s IN (%s, %s, %s)", inputs[0], inputs[1], inputs[2], inputs[0]), nil)
 	})
 }
+
+func BoolNot(yield Query) {
+	inputs := []string{"true", "false", "null"}
+	for _, input := range inputs {
+		yield(fmt.Sprintf("NOT %s", input), nil)
+	}
+}
```

Comment:
This is failing in CI `--- FAIL: TestMySQL/BoolNot (0.00s)`. 

```
=== RUN   TestMySQL/BoolNot
    comparison_test.go:201: evaluation type mismatch: eval=INT64 vs typeof=UINT64
        local: INT64(0)
        query: SELECT NOT true (SIMPLIFY=false)
    comparison_test.go:201: evaluation type mismatch: eval=INT64 vs typeof=UINT64
        local: INT64(1)
        query: SELECT NOT false (SIMPLIFY=false)
    --- FAIL: TestMySQL/BoolNot (0.00s)
```

After running it locally it seems like there is type mismatch between MySQL and my local version.

Reply:
I removed this commit from this PR since this was merge on main already